src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,349 @@
+which can be downloaded, stored by the browser or displayed in an `img` element.
+This method uses the highest available photographic camera resolution.
+* `grabFrame()` returns an `ImageBitmap` object,
+[a snapshot of live video](https://www.w3.org/TR/image-capture/#dom-imagecapture-grabframe),
+which could (for example) be drawn on a canvas and then post-processed to
+selectively change color values. Note that the `ImageBitmap` will only have the
+resolution of the video source — which will be lower than the camera's
+still-image capabilities.
+* `getPhotoCapabilities()` returns a `PhotoCapabilities` object that provides
+access to available camera options and their current values.
+* `setOptions()` is used to configure
+[camera settings](http://www.w3.org/TR/image-capture/#photosettings) such as
+zoom, white balance or focus mode.
+
+The Image Capture API gets access to a camera via a `MediaStream` from `getUserMedia()`:
+
+    navigator.mediaDevices.getUserMedia({video: true})
+      .then(gotMedia)
+      .catch(error => console.error('getUserMedia() error:', error));
+
+    function gotMedia(mediaStream) {
+      const mediaStreamTrack = mediaStream.getVideoTracks()[0];
+      const imageCapture = new ImageCapture(mediaStreamTrack);
+      console.log(imageCapture);
+    }
+
+You can try out this code from the DevTools console.
+
+Note: To choose between different cameras, such as the front and back camera on
+a phone, get a list of available devices via the
+`MediaDevices.enumerateDevices()` method, then set `deviceId` in `getUserMedia()`
+constraints — as per the demo [here](https://webrtc.github.io/samples/src/content/devices/input-output/).
+
+You can use `takePhoto()` to get a still image and then set it as the `src` of
+an `<img>`:
+
+    imageCapture.takePhoto()
+      .then(blob => {
+        image.src = URL.createObjectURL(blob);
+      })
+      .catch(error => console.error('takePhoto() error:', error));
+
+Use `grabFrame()` to get data for a frame of video and then draw
+it on a `<canvas>`:
+
+    imageCapture.grabFrame()
+      .then(imageBitmap => {
+        canvas.width = imageBitmap.width;
+        canvas.height = imageBitmap.height;
+        canvas.getContext('2d').drawImage(imageBitmap, 0, 0);
+      })
+      .catch(error => console.error('grabFrame() error:', error));
+
+You can try this out in the demo below — if you're using a browser that supports the Image Capture API:
+
+* The `<video>` displays a stream from a camera via `getUserMedia()`.
+
+* If you have more than one camera, you can select the video source.
+
+* Tap or click the **GRAB FRAME** button to draw a frame from the video on a
+`<canvas>`.
+
+* Tap or click the **TAKE PHOTO** button to call `takePhoto()` and display the
+result in an `<img>`.
+
+* If the camera you are using supports zoom, a zoom slider will be displayed.
+
+Several variables from the demo are available from the console:
+
+* **constraints**
+* **imageCapture**
+* **mediaStream**
```

Comment:
Since these are set off in a bulleted list, you probably don't need them to be bold.